### PR TITLE
Add Welcome bot

### PR DESCRIPTION
**Pull request by waimin314**
Add Welcome bot based on the Example Welcome bot. I will add the tests soon. Do let me know if I can improve in any way. thanks :)

### CODE REVIEW

File: src/bots/Welcome.ts [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  returningMember: 'Welcome back',
+};
+
+bot.on('new_chat_members', ctx => {
+  const userRepo = getRepository(User);
+
+  ctx.message.new_chat_members.forEach(async member => {
+    const user = await userRepo.findOne(member.id);
+    if (user === undefined && !member.is_bot) {
+      const newUser = userRepo.create({
```
Comment (xModulus) [inline src/bots/Welcome.ts:33]: There's no need to add new user to DB. The ScriberBot does this before everything else happens.

Comment (waimin314) [reply src/bots/Welcome.ts:33]: Thanks. I will fix it and submit again :)

Comment (waimin314) [reply src/bots/Welcome.ts]: submitted without adding the User to the database. added tests as well

**Comment by ruqqq** [comment]
Looks good!

Just some minor comments (but no rectification needed):
- Bot cannot see other bots message (I'll remove the commented code post merge)
- Unsure why generator is used in the test, perhaps can share more in our Telegram chat!

Merging.

**Pull request merged**